content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -286,7 +286,7 @@ To cut a release:
 
 [source,shell]
 ----
-git checkout master
+git checkout $(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
```

Comment:
I like the idea very much of telling the user to checkout the primary branch of the repository, without assuming the name of that primary branch.  We have several different primary branches already in use within the top 250 most popular plugins.  Those include "master", "main", and "develop" at a minimum.  Several of the repositories in the top 250 have branches using two or more of those names.

I think that we need to advise the user that this is specific to Unix or requires that they perform their release on Windows from inside a "git bash" window.

I don't know of a reasonable way to make that technique work from a Windows command prompt or from a Windows Powershell.  I think that a comment in the script would be enough to satisfy my concern.

This also assumes that the plugin maintainer has a remote named "origin" that points to the destination repository.  If I use `gh repo fork`, it configures a remote named "upstream" that would be the destination repository.  I assume we don't need to handle that case in the instructions.

```suggestion
# Checkout the primary branch of the repository
git checkout $(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
```